### ISSUE TITLE
When buying an album, 1st line is always written in USD

### ISSUE THREAD

**Issue by nicoalto** [funmusicplace/mirlo#810]
Hi,

When buying an album, the 1st line price is always written in USD, even though the artist is in another currency (EUR, etc.) For example my artist page is in EUR, but 1st line is shown in USD and only below EUR. Doing the maths shows that both prices seem to be in EUR (93% of 5 EUR = 4,65 EUR). Same issue with a Canadian artist showing USD/CAD. Is this a bug?

Thanks!
Nicolas

![Capture d’écran 2024-08-17 112157](https://github.com/user-attachments/assets/6ad18182-45fa-4fd1-a1fd-ddf1a5fca464)
Example link: https://mirlo.space/nicolas-hussein/release/compositions-vocales

**Comment by simonv3** [comment]
Hmm, yeah this does feel like a bug. What's making it more curious is that it's showing up as euros for me: 

![image](https://github.com/user-attachments/assets/c177fcfb-3d62-44c9-bc4e-aff45186552c)

**Comment by simonv3** [comment]
Okay I think what's going on here is that the currency defaults to USD as display if it is missing from the information. I'm thinking of removing the currency default on the money renderer component and hopefully that fixes it?

**Comment by simonv3** [comment]
gonna mark it as fixed, lmk if you're still seeing it

**Comment by nicoalto** [comment]
Still seeing it today...
![Capture d’écran 2024-09-14 130023](https://github.com/user-attachments/assets/364fa325-6e37-4b54-9153-812c67c2ef66)

**Comment by simonv3** [comment]
@nicoalto wondering if this is a caching issue--are you seeing this in private mode too?

**Comment by nicoalto** [comment]
Hi, yes I see this too in Firefox private mode...

**Comment by simonv3** [comment]
@nicoalto just confirming--still seeing this?

**Comment by nicoalto** [comment]
Yes... Firefox -> USD, Microsoft Edge -> EUR... (screenshots attached)

![Image](https://github.com/user-attachments/assets/3d95b050-b70f-495a-811e-f8aad4cdde64)
![Image](https://github.com/user-attachments/assets/4df21a51-9f50-45bf-aadb-6112e6948123)

**Comment by simonv3** [comment]
That's some crazy caching happening in Firefox. I'll have to think about how / whether I can bust that.
